src/sagas/flavors.js

```diff
@@ -0,0 +1,68 @@
+// import dayjs from 'dayjs';
+// import nanoid from 'nanoid';
+import {
+  all,
+  call,
+  put,
+  takeLatest
+  // delay,
+  // take,
+  // takeEvery,
+  // select
+} from 'redux-saga/effects';
+
+import request from 'utils/request';
+import { actions, types } from 'reducers/flavors';
+import { actions as appActions } from 'reducers/application';
+
+// snake_cased variables here come from RFC 6749
```

Comment:
This comment and the one below it are likely from a different file and unrelated.